### DISCUSSION TITLE
Repository

### DISCUSSION THREAD

**Discussion by IamGhost76x** [community/community#33531] [Other Feature Feedback, Questions, & Ideas]
I have seen my wife acting funny, so i look on her computer and find github.  I saw she made a repository using Ubuntu and helm v3.  Ill be honest, shes 11 years younger than me and i didn't even have internet in my school when i graduated.  She has had computers in her hands since she was born.  Woth all that said i think she is spying on me and when  i confront her about what im seeing she denies it.  I can read a bit what's going on by unicoding some of the garb. But, i can only read what i see.  Ive been checking in and im still nowhere.  Does anyone have a clue?

**Comment by IamGhost76x** [comment]
Any answers?

**Comment by radar-blip** [comment]
Possible paranoia.

**Reply by IamGhost76x** [reply]
That's what she said but I have the repository with her nickname and email address.

**Reply by radar-blip** [reply]
You're out-voted. Two to one. 😄
